### DISCUSSION TITLE
request: Missing function to split signature (present in Ethers)

### DISCUSSION THREAD

**Discussion by m1guelpf** [wevm/viem#932] [Idea / Feature Request] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.4.1

### Current Behavior

There's no provided function to split a signature into its components.

### Expected Behavior

When migrating from Ethers, there exists an alternative for the [`splitSignature`](https://github.com/ethers-io/ethers.js/blob/f97b92bbb1bde22fcc44100af78d7f31602863ab/packages/bytes/src.ts/index.ts#L333) function.

### Steps To Reproduce

N/A

### Link to Minimal Reproducible Example (StackBlitz, CodeSandbox, GitHub repo etc.)

N/A

### Anything else?

N/A

**Comment by jxom** [comment, marked as answer]
https://github.com/wagmi-dev/viem/discussions/458#discussioncomment-5842564

Should make a util for this. Open to PRs!
